src/AppInstallerCLICore/PortableInstaller.cpp

```diff
@@ -107,8 +107,9 @@ namespace AppInstaller::CLI::Portable
         }
         else if (fileType == PortableFileType::Directory)
         {
-            AICLI_LOG(Core, Info, << "Moving directory to: " << filePath);
-            Filesystem::RenameFile(entry.CurrentPath, filePath);
+            // Copy directory instead of renaming as there is a known issue with renaming across drives.
+            AICLI_LOG(Core, Info, << "Copying directory to: " << filePath);
+            std::filesystem::copy(entry.CurrentPath, filePath, std::filesystem::copy_options::overwrite_existing | std::filesystem::copy_options::recursive);

```

Comment:
I would really rather that we do something like:
```
if (IsSameVolume(current, target)
  Rename
else
  Copy
```
Renaming is just *so* much more efficient when it can be done that I think it is worth the extra effort.

Reply:
Added Filesystem::IsSameVolume to check for this and only copies when false.